_publications/2022-08-16-portmaster-0.9-new-network-monitor.md

```diff
@@ -0,0 +1,85 @@
+---
+# possible values: blog video podcast
+category: blog
+layout:   blog
+title: "Portmaster v0.9: New Network Monitor"
+date: 2022-08-16
+author: daniel
+custom_thumbnail_name: placeholder-announcement
+portmaster_plug: true
+portmaster_update: true
+---
+
+### New Network Monitor with Graphs
+
+Many of you have been waiting for this, and finally it is here: The new network monitor with elaborate search, filter and grouping support - with beautiful graphs everywhere!
```

Comment:
Many of you have been asking for this

I think waiting a bit pretentious, I would rather go with asking